tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -60,12 +60,12 @@ When(
 )
 
 When(
-  /^"([^"]*)" (copies|moves) the following (resource|resources)$/,
+  /^"([^"]*)" (copies|moves) the following resource(?:s)?( using (keyboard|drag-drop|sidebar))?(?: panel)?$/,
```

Comment:
```suggestion
  /^"([^"]*)" (copies|moves) the following resource(?:s)?( using (keyboard|drag-drop|sidebar panel))?$/,
```

Reply:
passing panel is not nice thing in my view

Reply:
why do you think so? panel is the part of sidebar only
You can write `sidebar panel` but not `keyboard panel` or `drab-drop panel`

Reply:
if we use a panel with a sidebar then we have to remove it later on which means more code. I think we can use general sense there that panel will be used with sidebar

Reply:
ahhh!
Did you check that `method` gets `keyboard|darg-drop|sidebar`, I am doubting the regex

Reply:
😅 working fine